.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -76,7 +90,7 @@ module.exports = (actionInfo) => {
           require('console').log(`Skipping ${pkgDataPath}`)
           continue
         }
-        const pkgData = require(pkgDataPath)
+        const pkgData = await fse.readJSON(pkgDataPath)
```

Comment:
we can use JSON.parse + fs.readFile here, we can drop fs-extra here

Reply:
@huozhi Done in [`a9d7f69` (#56525)](https://github.com/vercel/next.js/pull/56525/commits/a9d7f6971ada10e4c7cd4799e4cf386b37a8c031)

But do notice since we are also using `fs-extra`'s `copy` method, we can't drop `fs-extra` entirely.